Android/islandRush/app/src/main/java/com/quinstedt/islandRush/BrokerConnection.java

```diff
@@ -143,6 +222,66 @@ public void messageArrived(String topic, MqttMessage message) throws Exception {
                         String messageMQTT = message.toString();
                         setActualSpeedFromString(actualSpeed, messageMQTT);
                         Log.i(TAG, "Car speed: " + messageMQTT);
+                    }else if (topic.equals(playerName1)) {
```

Comment:
Whenever you have variables suffixed with an incrementing number, you might want an associative container instead? Think about what happens if you want 20 players in the future.

Reply:
By associative container, do you mean having key-value containers like in maps ?